tests/api2/test_stig.py

```diff
@@ -76,6 +62,56 @@ def two_factor_full_admin(two_factor_enabled, unprivileged_user_fixture):
         call('privilege.update', privilege[0]['id'], {'roles': []})
 
 
+def do_stig_auth(c, user_obj, secret):
+    resp = c.call('auth.login_ex', {
+        'mechanism': 'PASSWORD_PLAIN',
+        'username': user_obj.username,
+        'password': user_obj.password
+    })
+    assert resp['response_type'] == 'OTP_REQUIRED'
+    assert resp['username'] == user_obj.username
+
+    resp = c.call('auth.login_ex', {
+        'mechanism': 'OTP_TOKEN',
+        'otp_token': get_2fa_totp_token(secret)
+    })
+
+    assert resp['response_type'] == 'SUCCESS'
+
+
+@pytest.fixture(scope='module')
+def setup_stig(two_factor_full_admin):
+    """ Configure STIG and yield admin user object and an authenticated session """
+    user_obj, secret = two_factor_full_admin
+
+    # Create websocket connection from prior to STIG being enabled to pass to
+    # test methods. This connection will have unrestricted privileges (due to
+    # privilege_compose happening before STIG).
+    #
+    # Tests validating what can be performed under STIG restrictions should create
+    # a new websocket session
+    with product_type('SCALE_ENTERPRISE'):
+        with set_fips_available(True):
+            with client(auth=None) as c:
+                # Do two-factor authentication before enabling STIG support
+                do_stig_auth(c, user_obj, secret)
+                c.call('system.security.update', {'enable_fips': True, 'enable_gpos_stig': True}, job=True)
+                aal = c.call('auth.get_authenticator_assurance_level')
+                assert aal == 'LEVEL_2'
+
+                try:
+                    yield {
+                        'connection': c,
+                        'user_obj': user_obj,
+                        'secret': secret
+                    }
+                finally:
+                    c.call('system.security.update', {'enable_fips': False, 'enable_gpos_stig': False}, job=True)
+
+
+# The order of the following tests is significant. We gradully fixtures that have module scope
```

Comment:
If you wish yous can 'gradully' fix the typos.
Enough cheeky business.  Maybe:
`The order of the following tests are significant. We gradually add fixtures that have module scope`

I do love comments!